canonical-web-and-design/hiring.canonical.com
# Add parameter per_page to 500 since it defaults to 100

# Summary

The default number of applications returned is 100 per page by [the API.](https://developers.greenhouse.io/harvest.html#get-list-applications), updating the default to 500 since some jobs get a lot of tracktion.

# QA

- Connect with an account that has a lot of jobs
- http://0.0.0.0:8501/stages/applications-review
- Make sure the data is correct (chat with @tbille )
